GERTi/GERTiClient.lua

```diff
@@ -26,23 +26,83 @@ if (component.isAvailable("tunnel")) then
 end
 
 -- addresses
-local eAddress = 0
-local iAddress = 0
+
+-- This only matters to the "GERTi" API way down below. Should be int - see startup procedure.
+-- It is updated by handler["RegisterComplete"] if the message concerns this node.
```

Comment:
Please remove lines 30 and 31

Reply:
These are comments explaining that iAddress is only used by the GERTi API and not by general routing.

Reply:
They can stay for now, but I will probably reword them